superset-frontend/src/views/CRUD/chart/ChartCard.tsx

```diff
@@ -112,10 +110,12 @@ export default function ChartCard({
       )}
       {canEdit && (
         <Menu.Item
-          data-test="chart-list-edit-option"
+          data-test="chart-list-explore-option"
           role="button"
           tabIndex={0}
-          onClick={() => openChartEditModal(chart)}
+          onClick={() => {
+            window.location.href = chart.url;
+          }}
```

Comment:
Can we make this an actual `<a href>` instead of URL redirect via `location.href`? A lot of people uses `Cmd + Click` to open links in background tabs, using redirects in `onClick` wouldn't work for this use case.

Plus it should probably open in a new window.